lambdas/authorizationcode/src/main/java/uk/gov/di/ipv/cri/passport/authorizationcode/AuthorizationCodeHandler.java

```diff
@@ -174,6 +184,35 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private AuditEvent createAuditEventRequestSent(
+            String userId, DcsPayload dcsPayload, String clientId) {
+
+        PassportCheckDao passportCheckDao =
+                new PassportCheckDao(
+                        UUID.randomUUID().toString(), dcsPayload, null, userId, clientId);
+
+        VerifiableCredential vc = VerifiableCredential.fromPassportCheckDao(passportCheckDao);
```

Comment:
Do we need all this PassportCheckDao and VC stuff here? Do we not have all the data we want to include in the audit event just from the DcsPayload?

Might be simpler to just load the values straight from that and avoid constructing another VC?

Reply:
It was looking like it would be a big copy paste to take 90% of the vc generation code to do the same thing. I'd like to refactor this to make all the auditing much nicer, but thought this would work for now.